src/Address.svelte

```diff
@@ -95,7 +95,7 @@
   {/if}
   <div class="wrapper">
     {#if addressType === AddressType.Org}
-      <a use:link href={`/${nameOrAddress}`}>{addressLabel}</a>
+      <a use:router.link href={`/${nameOrAddress}`}>{addressLabel}</a>
```

Comment:
This seems weird, I would prefer to use the exported function as previously.. 😕

Reply:
I kind of want to get rid of the `link` action eventually, it's very awkward to use together with anchor tags, because Svelte doesn't like empty `href`s and the browser styling is off if we omit the `href`. But maybe that's a follow-up thing. Will restore the imports for now.

Reply:
3780790d1c8d9e807970c93365287db004ac422c